File: lua/1042.lua
local function sort(arr)
    for i = 1, #arr - 1 do
        for j = 1, #arr - i do
            if arr[j] > arr[j + 1] then
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
            end
        end
    end
    return arr
end

local l = {}
local old = {}

l[1] = io.read("*n")
l[2] = io.read("*n")
l[3] = io.read("*n")

for i = 1, #l do
    old[i] = l[i]
end

l = sort(l)

for i = 1, #l do
    print(l[i])
end

print()

for i = 1, #old do
    print(old[i])
end

File: go/1042.go
package main

import "fmt"

func sort(l [3]int) [3]int {
	for i := 0; i < len(l)-1; i++ {
		for j := 0; j < len(l)-i-1; j++ {
			if l[j] > l[j+1] {
				l[j], l[j+1] = l[j+1], l[j]
			}
		}
	}
	return l
}

func main() {
	var l, old [3]int
	fmt.Scanf("%d", &l[0])
	fmt.Scanf("%d", &l[1])
	fmt.Scanf("%d", &l[2])
	old = l
	l = sort(l)
	for i := 0; i < len(l); i++ {
		fmt.Printf("%d\n", l[i])
	}
	fmt.Printf("\n")
	for i := 0; i < len(old); i++ {
		fmt.Printf("%d\n", old[i])
	}
}
